### ISSUE TITLE
[How To] Shift right markdown list shortcut

### ISSUE THREAD

**Issue by deme0607** [HappenApps/Quiver#314]
Hi,

I want to know how to shift right markdown list by using shortcut.

For example, when I press enter key after typing a line, 
![image](https://cloud.githubusercontent.com/assets/1768650/11786447/0f3dcd14-a2ca-11e5-9650-6f74e8d0ff50.png)

I want to shift to right this second line list like following
![image](https://cloud.githubusercontent.com/assets/1768650/11786506/58686620-a2ca-11e5-8751-23b68fa5180f.png)

However, If I press tab key, just empty tab is inserted.
![image](https://cloud.githubusercontent.com/assets/1768650/11786528/79fb1de6-a2ca-11e5-83b1-c021754b3f7f.png)

Is there any way to shift right 2nd line?
I'm using vim key binding, and markdown cell.

**Comment by ylian** [comment]
Maybe not at the moment. Markdown support still has a lot of room for improvement.

**Comment by dhda** [comment]
With vim bindings, you can use the '>>' command to indent in normal mode.

**Comment by bruth** [comment]
If indented, `Shift+Tab` dedents a list item.

**Comment by mwean** [comment]
I would love to see this too

**Comment by kevanchristmas** [comment]
Me too - hope it gets fixed soon

**Comment by ghost** [comment]
+1

**Comment by ewellinger** [comment]
+1

**Comment by abMatGit** [comment]
+1

**Comment by Motutapu** [comment]
+1

**Comment by zachkirsch** [comment]
+1

**Comment by brandon-fryslie** [comment]
What's the status of this?  This is a real pain, this feature is pretty much the most basic level of Markdown support.

**Comment by blackwellops** [comment]
+1

**Comment by helloruiz** [comment]
+1. Having to move the cursor behind the asterisk in order to indent is beyond annoying.

**Comment by ethanfann** [comment]
I know you're super busy at the moment with the recent releases and I'm beyond excited to see Quiver getting some love again.

Once things calm down a bit, could we maybe get this looked at again? No support for indenting/outdenting with tab/shift-tab in the Markdown cell is the one thing keeping me on the Text Cell still.

**Comment by ylian** [comment]
@ethanfann Absolutely. Actually the next major effort is rewriting the Markdown and code editor. I will make sure this issue gets addressed during the rewrite.

**Comment by tkybpp** [comment]
Is there any update on this?

**Comment by devjona** [comment]
Please try this: 
* To _outdent_: `Ctrl + [` (with symbols)  `^ [`
* To _indent_  `Ctrl + ]` (with symbols) `^ ]` 
I remember hunting this down with trial and error because this shortcut isn't anywhere to be found in the nav menu, unless I've failed to find it.

The desired effect is:
1. I'm in a markdown cell and start a bulleted list with `*`
2. I press `Enter` and want to indent the new line I just created
3. I press `^ ]` and it indents:
<img width="445" alt="Screen Shot 2021-11-09 at 9 25 30 AM" src="https://user-images.githubusercontent.com/7801456/140952867-d086b06f-2994-45f0-ba6b-d125aaf4ea8e.png">

I'm on version: `3.2.7 (201)`
<img width="298" alt="Screen Shot 2021-11-09 at 9 18 37 AM" src="https://user-images.githubusercontent.com/7801456/140952627-a0de67fb-8205-401a-8403-5d30b45d61f9.png">

**Comment by devjona** [comment]
I think we're near to closing the issue, but I'd suggest adding this shortcut to the `Format` menu so that it's easy to find for current and future users.


<img width="252" alt="Screen Shot 2021-11-09 at 9 27 51 AM" src="https://user-images.githubusercontent.com/7801456/140953325-4a38bc37-9daf-4872-8d05-ec119c483732.png">
